### PR TITLE
Rewords the IC button

### PR DESCRIPTION
**Pull request by DominikPanic**
Slightly better version of  https://github.com/tgstation/tgstation/pull/44583

### CODE REVIEW

File: code/modules/admin/verbs/adminhelp.dm [hunk 1]
```diff
@@ -602,7 +602,7 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 		else if(tier == TICKET_ADMIN)
 			to_chat(initiator, "<font color='red' size='4'><b>- Adminhelp marked as IC! -</b></font><br>")
 			to_chat(initiator, "<font color='red'>Whatever your query was, you will have to find out using IC mean, the staff won't reveal anything relevant.</font>")
```
Comment (Rohesie) [inline code/modules/admin/verbs/adminhelp.dm]: Remove this line then?

Comment (DominikPanic) [reply code/modules/admin/verbs/adminhelp.dm]: It's still relevant.

**Comment by psykzz** [comment]
I would've expected the IC button to allow the admin to add option extra notes.

**Comment by DominikPanic** [comment]
> 
> 
> I would've expected the IC button to allow the admin to add option extra notes.

That's the same as bwoinking them. The buttons are meant to be one-click.

File: code/modules/admin/verbs/adminhelp.dm [hunk 2]
```diff
@@ -602,7 +602,7 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
...
-			to_chat(initiator, "<font color='red'>Your character will frequently die, sometimes without even a possibility of avoiding it. Events will often be out of your control. No matter how good or prepared you are, sometimes you just lose.</font>")
+			to_chat(initiator, "<font color='red'>Your issue has been determined by an administrator to be an in character issue and does not require their intervention at this time. For further resolution you should pursue options that are in character and within the rules.</font>.")
```
Comment (Rohesie) [inline code/modules/admin/verbs/adminhelp.dm]:
```suggestion
			to_chat(initiator, "<font color='red'>Your issue has been determined by an administrator to be in-character and does not require their intervention at this time, nor may they solve it without out-of-character intrusion into the round. You should find out a solution to the problem in the game, according to the rules.</font>")
```
Somewhat sleep-deprived, so no need to take it verbatim. Feel free to improve or ignore it.

**Pull request merged**